### PR TITLE
Colombia

### PR DESCRIPTION
**Pull request by MiguelARey**
Please check if everything is alright.
Thankyou!

### CODE REVIEW

File: colombia.html [hunk 1]
```diff
@@ -0,0 +1,313 @@
...
+            <li><a href="#place">Places</a></li>
+            <li><a href="#festival">Festival</a></li>
+            <li><a href="#food">Gastronomy</a></li>
+            <li><a href="#travel">Travel</a></li>
+        </ul>
+        <aside>
+            <i class="fa-solid fa-earth-americas"></i>
+        </aside>
+        <ul>
+            <li><a href="home.html">HOME</a></li>
```
Comment (Dayhun) [inline colombia.html]: You have to change it to 'href='index.html'

File: colombia.html [hunk 2]
```diff
@@ -0,0 +1,313 @@
...
+            <li><a href="brazil.html">BRAZIL</a></li>
+            <li><a href="colombia.html">COLOMBIA</a></li>
+            <li><a href="korea.html">KOREA</a></li>
+        </ul>
+        <details>
+            <summary class="fa-solid fa-bars"></summary>
+            <article>
+                <ul>
+                    <li><a href="#intro">Welcome</a></li>
```
Comment (Dayhun) [inline colombia.html]: Also you have to change to your own nav titles.

File: colombia.html [hunk 3]
```diff
@@ -0,0 +1,313 @@
...
+                    <li><a href="#summaryBrazil">About Brazil</a></li>
+                    <li><a href="#summaryColombia">About Colombia</a></li>
+                    <li><a href="#summaryKorea">About Korea</a></li>
+                </ul>
+                <ul>
+                    <li><a href="home.html">HOME</a></li>
```
Comment (Dayhun) [inline colombia.html]: index.html

**Review by Dayhun — COMMENTED** [review]
Check my review! Thanks.

File: colombia.html (hunk 1 above)
Comment (MiguelARey) [reply colombia.html]: fixed!

File: colombia.html (hunk 2 above)
Comment (MiguelARey) [reply colombia.html]: fixed!

File: colombia.html (hunk 3 above)
Comment (MiguelARey) [reply colombia.html]: fixed!

**Pull request merged**